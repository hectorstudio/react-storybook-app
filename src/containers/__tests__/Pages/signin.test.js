import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import SignIn from '../../Pages/signin';

describe('SignIn Page', () => {
  it('renders with default props', () => {
    const wrapper = shallow(<SignIn />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
