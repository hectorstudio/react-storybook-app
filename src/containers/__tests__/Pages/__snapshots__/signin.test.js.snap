// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn Page renders with default props 1`] = `
<div>
  <div>
    SignIn Page
  </div>
  <div>
    <input
      onChange={[Function]}
      type="text"
      value=""
    />
    <br />
    <input
      onChange={[Function]}
      type="password"
      value=""
    />
    <br />
    <button
      onClick={[Function]}
      type="button"
    >
      Sign In
    </button>
    <br />
    <Link
      to="/signup"
    >
      Try SignUp
    </Link>
  </div>
</div>
`;
